extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -203,7 +203,6 @@ public function loginDataProvider() {
 			'load' => null,
 			'loadFromDatabase' => null,
 			'checkPassword' => true,
-			'checkTemporaryPassword' => false,
 			'params' => array(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "params" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)